### ISSUE TITLE
Analytics: visibility tracking doesn't work with multi time constraints

### ISSUE THREAD

**Issue by zhouyx** [ampproject/amphtml#12761]
Visibility tracking in PROD doesn't work if both `continousTimeMin` and `totalTimeMin` are defined. 

For example. 
```
"pageview": {
    "on": "visible",
    "request": "base",
    "visibilitySpec": {
        "continuousTimeMin": 200,
        "totalTimeMin": 500
      }
}
```

**Comment by zhouyx** [comment]
Introduced by #12316, fixed by #12752

**Comment by zhouyx** [comment]
confirmed the fix is in prod.
